release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -404,6 +404,53 @@ In {product-title} {product-version} and later, you can edit the baseboard manag
+* node-exporter to 1.8.0
+* Prometheus to 2.52.0
+* Prometheus Operator to 0.73.2
+* Thanos to 0.35.0
+
+[id="ocp-4-16-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* Added the `ClusterMonitoringOperatorDeprecatedConfig` alert to monitor when the Cluster Monitoring Operator configuration uses a deprecated field.
+* Added the `PrometheusOperatorStatusUpdateErrors` alert to monitor when the Prometheus Operator fails to update object status.
+
+[id="ocp-4-16-monitoring-metrics-server-to-access-metrics-api-ga"]
+==== Metrics Server component to access the Metrics API general availability (GA)
+
+Metrics Server component is now generally available and automatically installed instead of the deprecated Prometheus Adapter. Metrics Server collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs, which frees the core platform Prometheus stack from handling this functionality. For more information, see xref:../observability/monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#metricsserverconfig[MetricsServerConfig] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-16-monitoring-new-role-to-allow-read-only-access-to-alertmanager-api"]
+==== New monitoring role to allow read-only access to the Alertmanager API
+
+This release introduces a new `monitoring-alertmanager-view` role to allow read-only access to the Alertmanager API in the `openshift-monitoring` project.
+
+[id="ocp-4-16-monitoring-vpa-metrics-available-in-kube-state-metrics-agent"]
+==== VPA metrics are available in the kube-state-metrics agent
+
+Vertical Pod Autoscaler (VPA) metrics are now available through the `kube-state-metrics` agent. VPA metrics follow a similar exposition format like they did before being deprecated and removed from native support upstream.
+
+[id="ocp-4-16-monitoring-change-in-proxy-service-for-monitoring-components"]
+==== Change in proxy service for monitoring components
+With this release, the proxy service in front of Prometheus, Alertmanager, and Thanos Ruler has been updated from OAuth to `kube-rbac-proxy`. This change might affect service accounts and users accessing these API endpoints without the appropriate roles and cluster roles.
+
+[id="ocp-4-16-monitoring-change-in-how-prometheus-handles-duplicate-samples"]
+==== Change in how Prometheus handles duplicate samples
+With this release, when Prometheus scrapes a target, duplicate samples are no longer silently ignored, even if they have the same value. The first sample is accepted and the `prometheus_target_scrapes_sample_duplicate_timestamp_total` counter is incremented, which might trigger the `PrometheusDuplicateTimestamps` alert. A user can refer to the runbook that is linked to the alert for help on how to resolve the issue.
```

Comment:
`A user can refer to the runbook that is linked to the alert for help on how to resolve the issue`?
do you mean the runbook_url is added to `PrometheusDuplicateTimestamps` alert? but did not find the runbook_url 
```
$ oc -n openshift-monitoring get prometheusrules prometheus-k8s-prometheus-rules -ojsonpath='{.spec.groups[].rules[?(@.alert=="PrometheusDuplicateTimestamps")]}' | jq
{
  "alert": "PrometheusDuplicateTimestamps",
  "annotations": {
    "description": "Prometheus {{$labels.namespace}}/{{$labels.pod}} is dropping {{ printf \"%.4g\" $value  }} samples/s with different values but duplicated timestamp.",
    "summary": "Prometheus is dropping samples with duplicate timestamps."
  },
  "expr": "rate(prometheus_target_scrapes_sample_duplicate_timestamp_total{job=~\"prometheus-k8s|prometheus-user-workload\"}[5m]) > 0\n",
  "for": "1h",
  "labels": {
    "severity": "warning"
  }
}
```

Reply:
I see maybe you meant https://github.com/openshift/cluster-monitoring-operator/pull/2365 which would be merged soon

Reply:
Ohh I see, @machine424 Should we remove the last sentence as the above PR was still not merged?

Reply:
As I mentioned [here](https://issues.redhat.com/browse/OCPBUGS-35397?focusedId=24963680&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24963680), I was willing to make sure the code and doc are synced. If you want to add that sentence once the fix is merged, we can remove it.
Maybe we don't even need it (users will spontaneously look for the runbook)

Reply:
Thank you for clarification! I missed that last comment in the Jira :+1:  I think we can remove the last sentence as you suggest :+1:

Reply:
no, nothing else.

Reply:
Okay so based on the comments here and in Slack, I am not sure what we want to do here, do we want to add the sentence about the runbook back? (I thought the main thing here was to highlight the new improved behavior)

Reply:
no need to add the runbook phrase back.

Reply:
got it, thanks!
https://github.com/prometheus/prometheus/releases/tag/v2.52.0
```
[ENHANCEMENT] Scrape: Increment prometheus_target_scrapes_sample_duplicate_timestamp_total metric on duplicated series during one scrape. https://github.com/prometheus/prometheus/pull/12933
```
OCP 4.16 uses prometheus v2.52.0, included the enhancement
https://github.com/openshift/prometheus/blob/release-4.16/CHANGELOG.md
```
[ENHANCEMENT] Scrape: Increment prometheus_target_scrapes_sample_duplicate_timestamp_total metric on duplicated series during one scrape. #12933
```